src/clj/game/cards/operations.clj

```diff
@@ -2306,7 +2306,9 @@
               :prompt "Add Subliminal Messaging to HQ?"
               :yes-ability
               {:msg "add Subliminal Messaging to HQ"
-               :effect (effect (move card :hand))}}}]})
+               :effect (effect (reveal eid card)
```

Comment:
as this is an `async` function, you gotta either call it last or write something like `:effect (req (wait-for (reveal state side (make-eid state eid) card)) (move state side card :hand) (effect-completed state side eid)))`, and of course you gotta add `:async true` to the ability map.